liquidator/liquidator.js

```diff
@@ -284,8 +286,26 @@ class Liquidator {
 
       // Send the transaction or report failure.
       let receipt;
+
       try {
-        receipt = await liquidation.send(txnConfig);
+        // Make sure to keep trying with this nonce
+        const nonce = await this.web3.eth.getTransactionCount(this.account);
+
+        // Min Gas Price, with a max gasPrice of x4
+        const minGasPrice = parseInt(this.gasEstimator.getCurrentFastPrice(), 10);
+        const maxGasPrice = 4 * minGasPrice;
+
+        // Doubles gasPrice every iteration
+        const gasPriceScalingFunction = ynatm.DOUBLES;
+
+        // Receipt without events
+        receipt = await ynatm.send({
+          sendTransactionFunction: gasPrice => liquidation.send({ ...txnConfig, nonce, gasPrice }),
+          minGasPrice,
+          maxGasPrice,
+          gasPriceScalingFunction,
+          delay: 60000 // Tries and doubles gasPrice every minute if tx hasn't gone through
```

Comment:
What happens if the tx has gotten to the end of the 3 doubles and is _still_ not included?

Reply:
If after the last tx is sent, and the <delay> has passed, then a `reject` will be thrown. The transaction(s) will be stuck in mempool.

Reply:
One option to potentially include down the road is for the ynatm library to optionally attempt a cancellation transaction. Not a high priority, though.